__tests__/intergration.tests.js

```diff
@@ -0,0 +1,56 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("/api/topics", () => {
+  test("GET: 200 returns status 200", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+
+  test("GET: 200 returns an array of correct length with elements consisting of each topic", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const expected = [
+          {
+            description: "The man, the Mitch, the legend",
+            slug: "mitch",
+          },
+          {
+            description: "Not dogs",
+            slug: "cats",
+          },
+          {
+            description: "what books are made of",
+            slug: "paper",
+          },
+        ];
+        expect(response.body.topics.length).toBe(3);
+        expect(response.body.topics).toEqual(expected);
+      });
```

Comment:
This test is perfectly fine but you could make it more flexible by looping through the array and using a flexible jest matcher such as toMatchObject